tools/lint.js

```diff
@@ -74,12 +77,15 @@ async function dlintPreferPrimordials() {
 
   const chunks = splitToChunks(sourceFiles, `${execPath} run`.length);
   for (const chunk of chunks) {
-    const { success } = await Deno.spawn(execPath, {
+    const cmd = new Deno.Command(execPath, {
       args: ["run", "--rule", "prefer-primordials", ...chunk],
       stdout: "inherit",
       stderr: "inherit",
     });
-    if (!success) {
+    cmd.spawn();
```

Comment:
I don't believe `cmd.spawn()` is needed in this case, as you're not doing anything fancy with the output.